base/docs/helpdb.jl

```diff
@@ -9537,12 +9539,20 @@ Returns an iterator over substrings of `s` that correspond to the extended graph
 """
 graphemes
 
-doc"""
+keywords[symbol("@__FILE__")] = doc"""
+
     @__FILE__() -> AbstractString
 
-`@__FILE__` expands to a string with the absolute path and file name of the script being run. Returns `nothing` if run from a REPL or an empty string if evaluated by `julia -e <expr>`.
+`@__FILE__` expands to a string with the absolute path and file name of the script being run.
+Returns `"none"` if run from a REPL or command-line context.
```

Comment:
Yeah that was left from the proposed change. I'll fix the doc.
On Aug 28, 2015 7:35 PM, "Tony Kelman" <notifications@github.com> wrote:

> In base/docs/helpdb.jl
> <https://github.com/JuliaLang/julia/pull/12727#discussion_r38254310>:
>
> >      @__FILE__() -> AbstractString
> >
> > -`@__FILE__` expands to a string with the absolute path and file name of the script being run. Returns `nothing` if run from a REPL or an empty string if evaluated by `julia -e <expr>`.
> > +`@__FILE__` expands to a string with the absolute path and file name of the script being run.
> > +Returns `"none"` if run from a REPL or command-line context.
>
> @ihnorton <https://github.com/ihnorton> can you confirm? Just tried it
> and this looks wrong.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/JuliaLang/julia/pull/12727/files#r38254310>.
>